bot/utilities/Queue.ts

```diff
@@ -1,3 +1,5 @@
+import { ChannelAccount, TurnContext } from "botbuilder";
+
```

Comment:
I'm interested as to why `ChannelAccount` is imported here?

Reply:
Ah, I think the TS extension adds the imports automatically and did so here. Ended up deleting its use, however, didn't notice this remaining.